src/components/wallet/PageTransactions.vue

```diff
@@ -195,8 +196,13 @@ export default {
             ...this.loadedTransactions,
             ...result.transactionsV2
           ]
+          // avoid duplicate transactions
+          const filteredLoadedTransactions = uniqWith(
```

Comment:
nooooop. as mentioned several times `hash` is not unique

Reply:
true, woah, you are so right. I forgot about multi messages once more